File: programme/volume-of-cuboid/volume-of-cuboid.c
// Write a programme to find volume of a cuboid

#include <stdio.h>

int main()
{
    int length, width, height, Volume;

    // Input
    printf("Enter the length of cuboid : ");
    scanf("%d", &length);
    printf("Enter the width  of cuboid : ");
    scanf("%d", &width);
    printf("Enter the height of cuboid : ");
    scanf("%d", &height);

    // calculate and print volume
    Volume = length * width * height;
    printf("\nVolume of Cuboid : %d\n", Volume);

    return 0;
}

File: programme/volume-of-cuboid/volume-of-cuboid.js
// Write a programme to find volume of a cuboid

// Input: length, width, height
const l = 2;
const w = 3;
const h = 4;

console.log("Length of Cuboid : " + l);
console.log("Width  of Cuboid : " + w);
console.log("Height of Cuboid : " + h);

function volumeCuboid(l, w, h) {
  return l * w * h;
}

console.log("\nVolume of Cuboid : " + volumeCuboid(l, w, h));
